package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "lint": "eslint \"src/**/*.js\" \"test/**/*.js\" \"examples/**/*.js\"",
     "doc": "jsdoc src/Core/Commander/Interfaces/ApiInterface/*",
-    "doclint": "npm run doc -- -t templates/silent",
+    "doclint": "npm run doc -- -t ../minami",
```

Comment:
And the `doclint` can keep the `silent` template, it would suppress its output, as its goal is just to test that the jsdoc is correct. (hint: it's correct only in ApiGlobe. We need to reactivate it for every js files, which would require some work to correct invalid syntax)